docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-product-offer-prices-feature-walkthrough.md

```diff
@@ -13,8 +13,48 @@ To learn more about the feature and to find out how end users use it, see [Produ
 
 {% endinfo_block %}
 
-## Related Developer articles
+## Marketplace Product Offer Prices - Overview
+Product offer prices can be defined for each product offer by merchants. 
+If price has not defined for product offer, will be used price from concrete product which is tied to the product offer by default.
+Each price should have type. 
+The price type it is an entity to define what type of prices will be used for selling. 
+There are 2 price types defined in the Spryker system, it's ORIGINAL and DEFAULT. 
+You can add own price types and use it in your app.
+The price can have gross or net value which can be used based on a price mode selected by customer in Storefront. 
+You can have shop running in both modes and select net mode for business customer, for example. 
+Price also has currency and store assigned to it.
+To support product offer prices has been added *PriceProductOffer* table to connect *PriceProductStore* and *ProductOffer* tables.
+To store data about product offer prices has been added *ProductConcreteProductOfferPriceStorage*.
+The data updated, when creating/updating/deleting/activating/deactivating concrete products and when create price product offers. 
+Also for product offers has been added support volume prices. 
+So merchants can add volume prices for product offers and based on chosen quantity of product offers customer will see the price on the Storefront.
+Volume prices for product offers work the same as for products.
+
+To learn more about prices and volume prices, see: [Prices](https://documentation.spryker.com/docs/prices-overview), [Volume Prices](https://documentation.spryker.com/docs/volume-prices-overview)
+
+## Module dependency graph
+![Entity diagram](https://confluence-connect.gliffy.net/embed/image/f128877d-eb61-4d87-b1af-5f166eb45c45.png?utm_medium=live&utm_source=confluence)
```

Comment:
- please add optional modules
- PriceProductOfferStorage -> ProductOffer, PriceProductOfferDataImport -> ProductOffer, PriceProductOffer -> -> ProductOffer, 
- I thins it is important to show that PriceProduct depends on Product, what do you think? (in this case dependencies from other module to Product should be added)

Reply:
I'm not sure that we need this information.
We provided links to related articles, including product offers.